src/watchful/client.py

```diff
@@ -83,7 +91,14 @@ def await_port_opening(port: int, timeout_sec: int = 10) -> None:
     raise TimeoutError("Timed out waiting for Watchful to start")
 
 
-def spawn_cmd(cmd: str, env: str = None) -> int:
+def spawn_cmd(
+    cmd: str,
+    env: Union[
```

Comment:
Why not Optional[...] like the current style/other changes made?

Reply:
The `Optional[...]` ones were changed by a tool, and will need to be revisited (most of them are wrong, and being ignored in this patch). In this case, `spawn_cmd` is calling into `subprocess.Popen`, and specifying an `env`. This type now matches the type of `env` in `subprocess.Popen`.